### DISCUSSION TITLE
Possibility to change the subject of notifications

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#13803] [Feature Requests]
I think it would be a good idea to have the possibility to change the subject of the notifications sent, either in directus or via email, I think everything would be easier and more organized. In our case in email notifications when we want to search for the subject it is impossible, because all emails are the same. And when each user receives dozens of notifications a day it gets harder :)

what do you think of this situation?

![image](https://user-images.githubusercontent.com/81326296/172811675-0ca2e801-160d-4e60-96fe-34bc778b7a87.png)

![image](https://user-images.githubusercontent.com/81326296/172813448-cac454fc-10d2-48b5-9265-4b341ef54006.png)

I think the difficulty would be that in each collection we would have to select a field designated for the subject.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
